modules/View.js

```diff
@@ -353,7 +364,7 @@
       options = options || {};
       view.detach();
       this.registerTrackedView(view, options);
-      view.attach($el);
+      view.attach($el, options);
```

Comment:
If we wanted to duck-type identify it in attach, then we could construct the options object here that contains the $el.
```
  options.$el = $el;
  view.attach(options)
```